Frontend/CO.CDP.OrganisationApp/Pages/Registration/CompanyHouseNumberQuestion.cshtml.cs

```diff
@@ -67,7 +67,7 @@ public async Task<IActionResult> OnPost()
                 {
                     flashMessageService.SetFlashMessage(
                         FlashMessageType.Important,
-                        heading: string.Format(StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner, OrganisationIdentifier, OrganisationName),
+                        heading: StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner, OrganisationIdentifier, OrganisationName,
```

Comment:
Looks like some stray bits left over from the string.format

```suggestion
                        heading: StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner,
```

Reply:
Fixed.